tekton/ci/jobs/tekton-golang-coverage.yaml

```diff
@@ -1,7 +1,7 @@
-apiVersion: tekton.dev/v1beta1
-kind: Task
+apiVersion: tekton.dev/v1alpha1
+kind: StepAction
 metadata:
-  name: go-coverage
+  name: go-coverage-sa
```

Comment:
I don't think we need to change the name here, the type of resource is already declared by `kind`. 
Besides `sa` could be confused with `service account`

Reply:
fixed